src/js/controller/instream-adapter.js

```diff
@@ -177,6 +177,10 @@ define([
             }
         };
 
+        this.clearAutostartMute = function() {
+            _model.set('autostartMuted', false);
+        };
```

Comment:
Shouldn't setting the player's mute state to false do this (`player.setMute(false)`)?